code/game/objects/structures/tables_racks.dm

```diff
@@ -62,6 +62,8 @@
 /obj/structure/table/attack_hand(mob/living/user)
 	if(user.a_intent == INTENT_GRAB && user.pulling && isliving(user.pulling))
 		var/mob/living/pushed_mob = user.pulling
+		if(!Adjacent(pushed_mob) && !Adjacent(user))

```

Comment:
`&& pushed_mob.ckey != kevinz000`

Reply:
Tempting. Very tempting.

Reply:
Actually after looking at this the only adjacent check should be the user since that's how it functions currently (minus the teleporting obv).

Reply:
https://github.com/tgstation/tgstation/pull/16347